boards/arm/nucleo_l4r5zi/doc/index.rst

```diff
@@ -218,10 +218,28 @@ Ethernet over USB is configured as the default network interface (EEM)
 Programming and Debugging
 *************************
 
-Currently, the usual way of programming and debugging does not
-exist. This will hopefully be added to a future version of the SDK. In
-the meantime, the `STM32 ST-LINK utility`_ program can be used by
-connecting CN1 to your PC and selecting the zephyr.bin file to flash.
+Connect the Nucleo L4R5ZI to your host computer using the USB port.
+Then build and flash an application. Here is an example for the
+:ref:`hello_world` application.
+
+Run a serial host program to connect with your Nucleo board:
+
+.. code-block:: console
+
+   $ minicom -D /dev/ttyACM0
```

Comment:
should we add ``-b 115200``?

Reply:
This is a tool setting, on my side it's default tool setting already, I've nothing to do. Maybe this is the same for other users. I guess it's up to user to know the tool, don't you think?
Important is we mention default setting (115200 8N1) for zephyr console some lines above. 
This is the same for other boards.